build/sync/plan.yml

```diff
@@ -0,0 +1,51 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
+actions:
+  # - paths:
+  #     - build
+  #   actions:
+  #     - type: overwrite_directory
+  #       params:
+  #         create: true
+  - paths:
+      - Makefile
+      - .editorconfig
+    actions:
+      - type: overwrite_file
+        params:
+          create: true
+        # conditions:
+        #   - type: file_unaltered
+        #     params:
+        #       repo: plugin
+        #       reference-repo: template
+  - paths:
+      # - go.mod
+      # - webapp/webpack.config.js
+      - webapp/tsconfig.json
+      - webapp/.eslintrc.json
+      # - webapp/src/index.js
+      # - server/configuration.go
+      # - server/plugin.go
+      # - server/plugin_test.go
+      # - public/hello.html
+      - .circleci/config.yml
+      # - webapp/i18n/en.json
+      # - webapp/package.json
```

Comment:
Why not include `webapp/package.json`?

Reply:
I recommend including `package.json` and `webpack.config.js` in the plan, then resolving diffs manually afterwards. This worked pretty well for me https://github.com/mattermost/mattermost-plugin-todo/blob/f7f527b9ac4c4ef181b279aaa80e6b1f2840c020/build/sync/plan.yml

Reply:
New commit syncs `webapp/package.json` and `webpack.config.js`

performed the following, post sync, to get these required packages.

`npm install --save superagent` 
`npm install --save js-cookie`